src/zoid/buttons/util.js

```diff
@@ -189,9 +189,9 @@ export function applePaySession() : ?ApplePaySessionConfigRequest {
                 listeners.paymentauthorized({ payment });
             };
 
-            session.oncancel = () => {
+            session.addEventListener('cancel', () => {
```

Comment:
should we ignore this to prevent the prod change?

Reply:
is this even right? I don't think that method exists on session: https://braintree.github.io/braintree-web/current/ApplePay.htm

Reply:
yeah i think the rule blindly applies to `oncancel` which kinda stinks... let's ignore this for now. I'll push an update

Reply:
`unicorn/prefer-add-event-listener`